### DISCUSSION TITLE
Is there a plan to support datepicker for mobile view?

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#48726] [Q&A]
Hi there,

So far I have heard of lot of feedback about Datepicker not working in small space or is unclickable in a modal. Is there a plan to design datepicker for mobile view?

thanks

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design-mobile/

**Reply by AsuraKev** [reply]
Sorry as in. Mobile responsiveness for web. I heard that datepicker is getting a rewrite. Has that already happened?
